test/mocha/assertions/issueWithoutStatus.js

```diff
@@ -86,7 +86,7 @@ export function testIssueWithoutStatus({
         should.not.exist(verifiableCredential.proof.created);
       }
     });
-    it('issues a valid credential w/"@language"', async () => {
+    it('issues a valid credential w/"@language"+"@dir"', async () => {
```

Comment:
This would be better as a full keyword `@direction` for easier searching and to avoid confusion.